### ISSUE TITLE
Changing the bind port of the container is reporting the internal port on start instead

### ISSUE THREAD

**Issue by reanimatedmanx** [muchobien/pocketbase-docker#27]
## Steps to repro

1. Copy the docker compose config [from here ](https://github.com/muchobien/pocketbase-docker?tab=readme-ov-file#using-docker-compose-recommended)
2. Change the port binding to ` - '8080:8090'`
3. Observe that the port reported in the logs is not the correct one. 
![image](https://github.com/user-attachments/assets/a2a73360-7164-41ad-9cf2-e8fd565a1295)

Q: Is this fixable like at all?:)

**Comment by KROSF** [comment]
That's correct—the binding port doesn't affect the reporting port at all. However, I can add support for a custom listening port if you'd like.

**Comment by reanimatedmanx** [comment]
Yeah I think that would be great, with an option to override it using an env variable, that way, it would report in the logs correctly.
